pkg/agent/proxy/proxier.go

```diff
@@ -292,9 +292,9 @@ func (p *proxier) syncProxyRules() {
 	defer func() {
 		delta := time.Since(start)
 		if p.isIPv6 {
-			metrics.SyncProxyDuration.Observe(delta.Seconds())
-		} else {
 			metrics.SyncProxyDurationV6.Observe(delta.Seconds())
+		} else {
+			metrics.SyncProxyDuration.Observe(delta.Seconds())
```

Comment:
since this is a pretty minor bug fix, I'm ok with merging this as part of this PR